slices/bash.yaml

```diff
@@ -0,0 +1,10 @@
+package: bash
+
+slices:
+  bins:
```

Comment:
this is called `bins`, however it isn't bringing all the binaries from that package. If you don't want the other binaries, I'd recommend renaming the slice to something more specific (like the binary name maybe...`bash`?)

Reply:
Alright, I've added additional binaries including `rbash` and `clear_console`. However, I think we need to include `bashbug`.

Reply:
Y I would say so

Reply:
Added, thanks!